src/main/java/payapal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
```

Comment:
Overall, would be good to see a couple of unit testing showing base/edge-case scenarios for this feature.